specification.md

```diff
@@ -320,7 +320,7 @@ to poll the Server like it is done when using the HTTP transport (see below).
 ## Plain HTTP Transport
 
 The second supported transport for OpAMP protocol is plain HTTP connection. The
-Agent is an HTTP client and the Server is an HTTP client server. The Agent makes
+Agent is an HTTP client and the Server is an HTTP server. The Agent makes
```

Comment:
Not sure, but perhaps for consistency:
```suggestion
Agent is an HTTP client and the Server is an HTTP Server. The Agent makes
```

Reply:
I intentionally skipped this. I use capital "S" to denote the OpAMP Server. The phrase "HTTP server" refers to any server.